MvvmCross/iOS/iOS/Views/Presenters/Attributes/MvxRootPresentationAttribute.cs

```diff
@@ -3,6 +3,6 @@ namespace MvvmCross.iOS.Views.Presenters.Attributes
 {
     public class MvxRootPresentationAttribute : MvxBasePresentationAttribute
     {
-        public bool WrapInNavigationController { get; set; }
+		public bool WrapInNavigationController { get; set; } = true;
```

Comment:
I'm not sure about making this member `true` by default, Because by doing so, when a user tries to set a `TabBarController` or a `SplitViewController` as root, he will have to set this to `false` explicitly